src/main/java/lotto/domain/Prize.java

```diff
@@ -0,0 +1,29 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum Prize {
+
+    first(6, 20000000),
```

Comment:
```suggestion
    first(6, 20_000_000),
```
언더스코어를 이용하면 가독성을 좀 더 확보할 수 있습니다 :)